sources/packages/forms/src/form-definitions/studentfinancialinformationappeal.json

```diff
@@ -134,7 +134,8 @@
                     "properties": {},
                     "redrawOn": "calculatedTaxYear",
                     "lockKey": true,
-                    "hideLabel": false
```

Comment:
This doesn't seem to be the right appeal. It is `partnerInformationAndIncomeAppeal`.

![image](https://github.com/user-attachments/assets/5ca84a90-7189-45ea-825f-c37d94490aae)